### ISSUE TITLE
Upgrading heimdalld-v*-*64.deb deletes provided files

### ISSUE THREAD

**Issue by etam** [maticnetwork/heimdall#1029]
**Heimdall version**: 0.3.3, 0.3.4

**Environment**:
- **OS** (e.g. from /etc/os-release): Ubuntu 20.04
- **Install tools**: ?
- **Others**: ?

**What happened**:
```sh
$ ls /usr/bin/heimdall*
ls: cannot access '/usr/bin/heimdall*': No such file or directory
```

**What you expected to happen**: Files should exist

**Have you tried the latest version**: yes

**How to reproduce it** (as minimally and precisely as possible):
- Install heimdalld from deb with `sudo dpkg -i heimdalld-v0.3.3-amd64.deb`
- Upgrade to next version with `sudo dpkg -i heimdalld-v0.3.4-amd64.deb`

**Anything else we need to know**:
This is caused by the `postrm` ([this](https://github.com/maticnetwork/heimdall/blob/develop/packaging/templates/package_scripts/postrm.binary) or [this](https://github.com/maticnetwork/heimdall/blob/develop/packaging/deb/heimdalld/DEBIAN/postrm)) file, which contains
```
sudo rm -rf /usr/bin/heimdallcli
sudo rm -rf /usr/bin/heimdalld
```

This is similar to https://github.com/maticnetwork/bor/issues/946

**Comment by VAIBHAVJINDAL3012** [comment]
HI @etam , thanks for reporting this issue.
We will raise the PR to solve this soon.

**Comment by etam** [comment]
Please don't close this. A corresponding issues in bor were already resolved (https://github.com/maticnetwork/bor/issues/946 and https://github.com/maticnetwork/bor/issues/957).

**Comment by etam** [comment]
still not resolved

**Comment by etam** [comment]
still not resolved

**Comment by pratikspatil024** [comment]
Fixed in [PR#1064](https://github.com/maticnetwork/heimdall/pull/1064). Thanks.
